src/js/util/factory.js

```diff
@@ -0,0 +1,130 @@
+define([
+  'd3',
+  'lodash',
+  '../models/radar.js',
+  '../models/quadrant.js',
+  '../models/ring.js',
+  '../models/blip.js',
+  '../graphing/radar.js',
+  '../config.js',
+  '../graphing/config.js'
+], function(d3, _, Radar, Quadrant, Ring, Blip, GraphingRadar, configFuntion, config) {
```

Comment:
Just outta curiosity, what's d3 and _? :)

Reply:
they are both js libraries, d3 is a statistics javascript library which is a "open-source JavaScript library for visualizing data" and lodash (_) "makes JavaScript easier by taking the hassle out of working with arrays, numbers, objects, strings, etc." We refer to them by name after defining them in a config file.